sdk/helper/consts/fips.go

```diff
@@ -0,0 +1,9 @@
+//go:build !fips_140_3
+// +build !fips_140_3
```

Comment:
Only need the second of these, on the very first line.

Reply:
😞 I need to figure out the GoLand setting that keeps adding these back...